### DISCUSSION TITLE
Jan: Model switching while the server is running

### DISCUSSION THREAD

**Discussion by eckartal** [janhq/jan#3623] [Feature Ideas]
## Overview
Jan users start their own Jan servers with one model and they can't change the model while the server is running. We should support changing models remotely.

## Goals

- Allowing change of the active model from any client device (phone, laptop, etc.)
- Provide multiple model-switching methods to suit different use cases (API, UI, automatic?)
- Ensure minimal downtime during the model-switching process

**Comment by nvdg** [comment]
Whaaat how is this not a thing yet! I thought I could ditch LM studio but not having access to all models through the API is a deal breaker.
